### PR TITLE
refactor : change font in blog website

### PR DESCRIPTION
**Pull request by cp-mansi-d**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [ ] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Description**
- change fonr style in whole blog website.
- another some changes related font.
- change some font opacity in title section and also in sharing icons.
- also tried to used default tailwind classes ,if possible

### CODE REVIEW

File: components/posts/recommandedPosts.js [hunk 1]
```diff
@@ -38,11 +38,13 @@ export default function RecommandedPosts({ postData }) {
...
                       </div>
-                      <span className="font-medium">{post.authorName}</span>
+                      <span className="font-worksans-medium tracking-wide">
+                        {post.authorName}
+                      </span>
                     </div>
 
                     <div
-                      className="text-[1rem] sm:text-[1.1rem] md:text-[1.25rem] xl:text-[1rem] font-bold leading-[1.27rem] md:leading-[1.7rem] xl:leading-5 tracking-[0.06rem] xl:line-clamp-3"
+                      className="text-[1rem] sm:text-[1.1rem] md:text-[1.25rem] xl:text-[1.01rem] leading-[1.27rem] md:leading-[1.7rem] xl:leading-5 tracking-[0.01rem] xl:line-clamp-3 text-[#000000] font-medium font-worksans-medium"
```
Comment (cp-priyanka-g) [inline components/posts/recommandedPosts.js:47]: use default tailwind class for text-size.

File: pages/index.js [hunk 2]
```diff
@@ -302,9 +302,9 @@ export default function Home({ posts, status, categories, mixpanel }) {
                   <div className="flex justify-end mt-10">
                     <Link
                       href="/featured"
-                      className="relative rounded-full border-[1px] border-solid border-transparent bg-gradient-to-r from-[#f2709c] to-[#ff9472] hover:shadow-[inset_2px_1000px_1px_#fff] py-[0.5rem] font-bold text-white"
+                      className="relative rounded-full border-[1px] border-solid border-transparent bg-gradient-to-r from-[#f2709c] to-[#ff9472] hover:shadow-[inset_2px_1000px_1px_#fff] py-[0.5rem] font-worksans-semibold text-white"
```
Comment (cp-jagruti-a) [inline pages/index.js:305]: you can use border only here

File: components/posts/recommandedPosts.js (hunk 1 above)
Comment (cp-mansi-d) [reply components/posts/recommandedPosts.js:47]: This default tailwind class does not provide the same leading that I need with this text. i will used in website if any place default tailwind class work.

Comment (cp-priyanka-g) [reply components/posts/recommandedPosts.js:47]: there is class "text-base" can be used in place of text-[1rem]

Comment (cp-mansi-d) [reply components/posts/recommandedPosts.js:47]: that write but with that provide line-height=1.5rem and i need to **line-height=1.27rem**. so that

Comment (cp-priyanka-g) [reply components/posts/recommandedPosts.js:47]: if you use you own leading-size it will override the leading of default classs. so you can use it freely.

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**